pages/about.html

```diff
@@ -4,9 +4,100 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="styles/style.css" rel=Stylesheet>
 </head>
-<body>
+<body> 
+  <header class="about-header">
+    <nav class="ss-bg">
+      <ul id="menu" class="about-menu">
+        <li><a href="http://127.0.0.1:5502/pages/index.html"><h3>Home</h3></a></li>
+        <li><a href="http://127.0.0.1:5502/pages/about.html"><h3>About</h3></a></li>
```

Comment:
These links do not work. We should use relative paths to reference other pages, similar to what you have on line 16 to get to the assets. Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/